src/test/java/hw1/CommonConditions.java

```diff
@@ -1,19 +0,0 @@
-package hw1;
```

Comment:
1. Why did you delete files for the hw1 from the current Pull Request?
2. Why files of the hw1 exist in the current Pull Request?

Reply:
These files got here from the Master branch. And I deleted them because after the changes in the pom.xml without special plugin settings (with default settings), they began to run everything that seems like a test, I decided that it was better to leave only those related to the task, everything is still in the Master branch. :)